pkg/mhooks/payload.go

```diff
@@ -0,0 +1,57 @@
+package mhooks
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+)
+
+func NewPayload(requestBody io.Reader) (*Payload, error) {
+	var payload Payload
+	err := json.NewDecoder(requestBody).Decode(&payload)
+	if err != nil {
+		return nil, fmt.Errorf("decoding payload: %w", err)
+	}
+
+	return &payload, nil
+}
+
+type Payload struct {
+	EventID   string          `json:"eventID"`
+	EventType EventType       `json:"type"`
+	Data      json.RawMessage `json:"data"`
```

Comment:
Given the methods below, I can make this field private.

Reply:
Is there code we could copy from webhooks (and have webhooks import it from moov-go)?

Reply:
Good idea; this is essentially copied from `webhooks`. Once this is merged in; i'll update `webhooks` to reuse this + reuse the enums too. Will help us avoid drift.